gogio/iosbuild.go

```diff
@@ -353,13 +379,36 @@ func buildInfoPlist(bi *buildInfo) string {
 	<key>DTSDKBuild</key>
 	<string>16G73</string>
 	<key>DTSDKName</key>
-	<string>%s12.4</string>
+	<string>{{.Platform}}12.4</string>
 	<key>DTXcode</key>
 	<string>1030</string>
 	<key>DTXcodeBuild</key>
 	<string>10G8</string>
+    {{if .Schemes}}
+	<key>CFBundleURLTypes</key>
+	<array>
+	  {{range .Schemes}}
+	  <dict>
+		<key>CFBundleURLSchemes</key>
+		<array>
+		  <string>{{.}}</string>
+		</array>
+	  </dict>
+	  {{end}}
+	</array>
+    {{end}}
 </dict>
-</plist>`, appName, bi.appID, appName, bi.version, bi.version.VersionCode, platform, minIOSVersion, supportPlatform, platform)
+</plist>`)
+	if err != nil {
+		return "", err
```

Comment:
Any error here is a result of a programming mistake, so it should panic instead of returning.